src/controllers/accommodationController.js

```diff
@@ -11,6 +11,32 @@ const association = [
   }
 ];
 
+const reactToAccommodation = async (req, res, reactionType) => {
+  const { accommodationId } = req.params;
+  const { userId } = req.user;
+  try {
+    const result = await Reaction.findOne({
+      where: { accommodationId, userId }
+    });
+    if (!result) {
+      await Reaction.create({ reaction: reactionType, userId, accommodationId });
+    } else if (result.reaction === reactionType) {
+      await Reaction.destroy({
+        where: { accommodationId, userId }
+      });
+    } else {
+      await Reaction.update({ reaction: reactionType },
+        { where: { accommodationId, userId } });
+    }
+    const { count } = await Reaction.findAndCountAll({
+      where: { accommodationId, reaction: reactionType }
+    });
+    return successResponse(res, 200, `Total number of ${reactionType}: ${count}`);
```

Comment:
Your PR says the response is
```
{
    "status": 200,
    "message": "Total number of likes: 1"
}
```
but the actual response you're sending is:
```
{
    "status": 200,
    "message": "Total number of true: 1"
}
```
You're sending `true` instead of `like`. The same thing for `dislike`. You're sending `false`.
The problem is in this statement `Total number of ${reactionType}: ${count}`. The `reactionType` is originally sent as either `true` or `false`. So you need to do a mapping of `reactionType` to the actual word you want to send in your response.

Reply:
Fixed @eokwukwe